src/protobuf-net/Meta/MetaType.cs

```diff
@@ -2264,6 +2318,8 @@ void AppendReservations()
             }
         }
 
+        private static string GetWrappedSchemaTypeName(string schemaTypeName) => $"Wrapped{schemaTypeName}";

```

Comment:
`"Wrapped" + schemaTypeName` please - more direct

Reply:
just to learn: why is it better? I thought string concatenation is not a best option for performance - or do we rely on generated IL so much, that we will see something else there?

P.S. changed to yours proposition

Reply:
Ultimately, this is getting compiled (optimized) to the same thing ([example here](https://sharplab.io/#v2:D4AQzABCBMEMIQN4FgBQEMTZqkQEYAGCAMQHsyBjAQwBcAKA4gZ0oAsBTAW2oBUBPAA4cActS4cAlBAC8APmw5MAIgDqAJ2qDhAE2UQA1BFaceA4WIkBuRRlu4oRUhXU8GTY+258ho8VNk5LHQlDAASNU1tDh1EE29zPwkAX2UbELtUZKA==)) - so expressing it as a concat is more direct/accurate to what's happening :)

Reply:
Darn it I was just sharplabing!  In this case it works out the same, so: fine. There are many cases where it doesn't. Now: that doesn't mean we shouldn't use it! There are many cases where it simplifies code greatly, but then the question is "how often does this run, do I care about extra overheads?". The main reason I said to prefer simple concat is just to reduce mental load, so I don't even need to ask myself that question.